integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -50,20 +59,19 @@ def _is_exists(self, group: RESTObject) -> bool:
                 return True
         return False
 
-    def _create_group_webhook(self, group: RESTObject) -> None:
+    def _create_group_webhook(self, group: RESTObject, events: list[str] | None) -> None:
         logger.info(f"Creating webhook for {group.get_id()}")
```

Comment:
I would move this log message after the events resolution add add the events as part of the log

Reply:
done